templates/localgov-blogs-prev-next-block.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+* @file
+* Default theme implementation for the localgov_blogs_prev_next_block block.
+*/
+#}
+<nav class="localgov_blogs--navigation">
+  {% if previous_url %}
+    <a href="{{ previous_url }}" class="localgov-blogs--previous" rel="prev" aria-label="{{ 'Previous'|t }}: {{ previous_title }}">
+      <span class="fa-solid fa-chevron-left"></span>
```

Comment:
Let's not use font-awesome classes here, as we may not have that library installed on all sites.

Reply:
Yes, I see that, however if you don't nothing gets dsiaplyed, if you do that would mean not having to have a copy of the twig in custom theme.
Which is better/worse? I was taking a lead from localgov_guides: https://github.com/localgovdrupal/localgov_guides/blob/2.x/templates/guides-prev-next-block.html.twig

Reply:
Maybe this is okay as a base template that can be used on all sites (even ones not using LGD itself), but if so, we'd probably then create a duplicate of it in `localgov_base` like we did for the Guides Nav. You can see it here: https://github.com/localgovdrupal/localgov_base/blob/1.x/templates/block/guides-prev-next-block.html.twig

In that we are using icons that we have available in `localgov_base` and we then know all the sites using this template will have those icons (since they are supplied as SVG files in the theme). 

Adding ad-hoc `fa-*` classes will not give any output if the site in question does not also load Font Awesome. The same issue happens with some of our legacy Bootstrap classes that are in various places of the codebase.

Reply:
I think for base templates we shouldn't assume any other libaries are present and remove it, keep it just the link. 
Providing nice theming is then a responsibility of localgov_base, which most sites use as a base theme and will get the chevrons. Others can then custmise the template in their own theme or even just use a preprocess hook to add their own chevrons (BHCC has an SVG sprite we would reference).

Reply:
OK - I'll remove the Font Awesome from here.
Do we want to add theming to `localgov_base` or leave it for site devs to add their own?

Reply:
@AWearring if you just create an issue on `localgov_base` to add the blogs navigation template, I can take care of doing the actual work on it.